tests/test_converter.py

```diff
@@ -495,19 +507,22 @@ def test_operations_transformed_into_qubit_unitary(self, recorder):
         assert len(recorder.queue[1].params) == 0
         assert recorder.queue[1].wires == [0]
 
-        assert recorder.queue[2].name == 'U2'
-        assert len(recorder.queue[2].params) == 2
-        assert recorder.queue[2].params == [0.3, 0.4]
-        assert recorder.queue[2].wires == [0]
+    def test_operation_transformed_into_qubit_unitary(self, recorder):
+        """Tests loading a circuit with operations that can be converted,
+         but not natively supported by PennyLane."""
 
-        assert recorder.queue[3].name == 'U3'
-        assert len(recorder.queue[3].params) == 3
-        assert recorder.queue[3].params == [0.3, 0.4, 0.2]
-        assert recorder.queue[3].wires == [0]
+        qc = QuantumCircuit(3, 1)
 
-        assert recorder.queue[4].name == 'Toffoli'
-        assert len(recorder.queue[4].params) == 0
-        assert recorder.queue[4].wires == [0, 1, 2]
+        qc.ch([0], [1])
```

Comment:
Added a test case for an operation from Qiskit that has the ``to_matrix()`` method defined on them and is not in PL core

Reply:
nice!